src/utils/auth.js

```diff
@@ -10,6 +11,11 @@ export const pwConfirmError = document.querySelector("#pwConfirm-error");
 export const pwConfirmInput = document.querySelector("#password-confirm");
 export const pwConfirmToggle = document.querySelector("#pwConfirm-eyeIcon");
 
+export const checkLoginStatus = () => {
+  const tokens = getTokens();
+  if (tokens.accessToken) window.location.href = "folder.html";
+};
```

Comment:
토큰 유무를 확인하고 페이지를 이동하는 동작을 하는 함수인데, 함수명이 모호하네요.
추가로, 구조분해할당해서 사용하면 더 깔끔할 것 같습니다.
```suggestion
export const checkLoginStatus = () => {
  const { accessToken } = getTokens();
  if (accessToken) window.location.href = "folder.html";
};
```